addons/token-exchange/manifests/hub_role.yaml

```diff
@@ -0,0 +1,18 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: open-cluster-management:token-exchange:agent
+  namespace: {{ .ClusterName }}
+rules:
+- apiGroups: ["addon.open-cluster-management.io"]
+  resources: ["managedclusteraddons"]
+  verbs: ["get", "list", "watch", "update", "patch"]
+- apiGroups: ["addon.open-cluster-management.io"]
+  resources: ["managedclusteraddons/finalizers"]
+  verbs: ["update"]
+- apiGroups: ["addon.open-cluster-management.io"]
+  resources: ["managedclusteraddons/status"]
+  verbs: ["patch", "update"]
+- apiGroups: [""]
+  resources: ["secrets"]
```

Comment:
this only ensure that the agent can create secrets on cluster ns on hub, but not in other ns

Reply:
Good point. This should be a ClusterRole.